### DISCUSSION TITLE
FirebaseMessaging crashing Android Go

### DISCUSSION THREAD

**Discussion by fbuckeri** [firebase/flutterfire#12086] [General]
Guys, sorry for the English. I am using the google translator.

I am facing an issue regarding using FirebaseMessaging on Android Go.

Other versions of Android are working perfectly.

When the app starts, it goes through this call: **`await FirebaseMessaging.instance.subscribeToTopic('all');`**.

When executing this line the app simply crashes. There is no error, no progress. It simply freezes.

I tested by running Android Studio in Release, Debug and also installing both versions of the app on Android Go and the error is the same. Freezes on that line.

Has anyone ever experienced this.

I looked for something related here and couldn't find it. I apologize if this has already been addressed in another thread. I didn't really identify anything about it.

**Comment by GIGAM25** [comment]
same issue. can't find solution
